resources/home/dnanexus/utils/utils.py

```
@@ -0,0 +1,90 @@
+"""general and I/O utilities
+"""
+
+import pandas as pd
+import openpyxl
```

Ruff (0.8.2)

`openpyxl` imported but unused

Remove unused import: `openpyxl`

(F401)